doc/MANUAL.md

```diff
@@ -179,62 +320,53 @@ This option allows you to specify the filename for the report, rather than havin
 
-This option allows the CVE Binary Tool to produce a report in an alternate format. This is useful if you have other tools which only take a specific format. The default is `console` which pretty-prints the information.
-1. `--format csv` - prints in csv (comma separated) format.
-```
-libgcrypt,1.6.0,CVE-2017-9526,MEDIUM
-libgcrypt,1.6.0,CVE-2018-0495,MEDIUM
-libgcrypt,1.6.0,CVE-2018-6829,HIGH
+This option allows the CVE Binary Tool to produce a report in an alternate format. This is useful if you have other tools which only take a specific format. The default is `console` which prints category wise beautiful tables of CVEs on terminal.
+
+1.  `--format csv` - write output file in csv (comma separated) format.
+
+```csv
+vendor,product,version,cve_number,severity,remarks,comments
+haxx,curl,7.34.0,CVE-2014-0015,MEDIUM,Mitigated,
+haxx,curl,7.34.0,CVE-2014-0138,MEDIUM,NewFound,
+haxx,curl,7.34.0,CVE-2014-0139,MEDIUM,Unexplored,
 ```
-2. `--format json` - prints in json (javascript object notation) format.
+
+2.  `--format json` - write output file in json (javascript object notation) format.
+
 ```json
 [
-    {
-        "package": "libgcrypt",
-        "version": "1.6.0",
-        "cve_number": "CVE-2017-9526",
-        "severity": "MEDIUM"
-    },
-    {
-        "package": "libgcrypt",
-        "version": "1.6.0",
-        "cve_number": "CVE-2018-0495",
-        "severity": "MEDIUM"
-    },
-    {
-        "package": "libgcrypt",
-        "version": "1.6.0",
-        "cve_number": "CVE-2018-6829",
-        "severity": "HIGH"
-    }
+  {
+    "vendor": "haxx",
+    "product": "curl",
+    "version": "7.34.0",
+    "cve_number": "CVE-2014-0015",
+    "severity": "MEDIUM",
+    "remarks": "Mitigated",
+    "comments": ""
+  },
+  {
+    "vendor": "haxx",
+    "product": "curl",
+    "version": "7.34.0",
+    "cve_number": "CVE-2014-0138",
+    "severity": "MEDIUM",
+    "remarks": "NewFound",
+    "comments": ""
+  },
 ]
 ```
-3. `--format console` - prints in nice tabular format.
-```console
-+=================================================================+
-|   CVE Binary Tool Report Generated: 2020-04-29  10:04:06        |
-+=================================================================+
-
-+=================================================================+
-|   MODULE NAME      |  VERSION  |    CVE NUMBER      | SEVERITY  |
-+=================================================================+
-| bzip2              | 1.0.2     | CVE-2005-0953      | LOW       |
-+--------------------+-----------+--------------------+-----------+
-| bzip2              | 1.0.2     | CVE-2005-1260      | MEDIUM    |
-+--------------------+-----------+--------------------+-----------+
-| bzip2              | 1.0.2     | CVE-2008-1372      | MEDIUM    |
-+--------------------+-----------+--------------------+-----------+
 
-```
+3.  `--format console` - prints in nice colored tabular format.
+![format console](https://i.imgur.com/UwH6vA7.png)
```

Comment:
For accessibility, we should show a non-colourized version of this as alt text or caption or just below.

Reply:
Done :)